.ci/scripts/release/post-github.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd ${GOPATH}/src/github.com/zeebe-io/zeebe/clients/go/internal/embedded
+
+echo ${DEVELOPMENT_VERSION} > data/VERSION
+go-bindata -pkg embedded -o embedded.go -prefix data/
+
+git commit -am "chore(project): update version in go client"
```

Comment:
Nit: maybe we can copy the commit message the maven plugin use? "Prepare next development version" or something

Reply:
Sure, something like "chore(project): Prepare next development version (Go client)"? Otherwise, it might seem weird to have the "same" commit twice